lib/connector.js

```diff
@@ -218,6 +219,26 @@ Connector.prototype.id = function(model, prop) {
   return p && p.id;
 };
 
+/**
+ * Return the database mapping name of the property of the model if it exists.
+ * Otherwise return the property name.
+ * @param {String} model The target model name
+ * @param {String} prop The property name
+ */
+Connector.prototype.propMapping = function(model, prop) {
+  return SQLConnector.SQLConnector(model, prop);
```

Comment:
How does this perform a mapping? To me, it just looks like it is returning a constructor for SQL connector. What function is actually being called here? thx.

Reply:
Also, SQLConnector inherits (via mixins) all the functions (and possibly properties) of Connector via this line:
https://github.com/strongloop/loopback-connector/blob/master/lib/sql.js#L32
So, when `Connector.prototype.propMapping` is called, it calls SQLConnector(model,prop). So if SQLConnector inherits from Connector, SQLConnector will also contain propMapping. Is this what we want?

Reply:
These outdated code. Please review the latest changed code